### ISSUE TITLE
Accessibility issues on Taxonomy Picker

### ISSUE THREAD

**Issue by jayesh889** [pnp/sp-dev-fx-controls-react#837]
#### Version

Please specify what version of the library you are using: [    2.4.0     ]

#### Expected / Desired Behavior / Question
   1. When nothing is selected -  In fast pass run (Accessibility Insights for Web) reports failed label, failed button-name
   2. When something already selected - In fast pass run (Accessibility Insights for Web) reports failed aria required children, 
        failed label, failed button-name
  3. Not able to navigate through selected options in field by tabbing
  4. Text Failing in dark theme when click on icon

#### Observed Behavior
  1. No issue raised
  2. No issue raised
  3. Able to navigate through selected options in field by tabbing
  4. All Text should visible in dark theme when click on icon

**Comment by kunj-sangani** [comment]
Hi @jayesh889 

Can you please provide some screenshot as well as can you please provide some details steps how to replicate the issue

**Comment by jayesh889** [comment]
> Hi @jayesh889
> 
> Can you please provide some screenshot as well as can you please provide some details steps how to replicate the issue
Steps to follow -
1.First you need to add extension Accessibility insights for web on your browser
2.Click on fast pass under it
